website/versioned_docs/version-0.12.2/quick-start-guide.md

```diff
@@ -761,7 +761,6 @@ MERGE INTO tableIdentifier AS target_alias
 USING (sub_query | tableIdentifier) AS source_alias
 ON <merge_condition>
 [ WHEN MATCHED [ AND <condition> ] THEN <matched_action> ]
-[ WHEN MATCHED [ AND <condition> ] THEN <matched_action> ]
 [ WHEN NOT MATCHED [ AND <condition> ]  THEN <not_matched_action> ]
 
```

Comment:
Can we fix all the versions that support MERGE INTO?

Reply:
remove this line in all version of docs